lib/SparqlXmlParser.ts

```diff
@@ -132,6 +132,11 @@ export class SparqlXmlParser {
       }
       stack.pop();
     })
+    parser.on("cdata", text => {
```

Comment:
Since the listener is identical to the one below, can be just create it once, and attach it for both the `cdata` and `text` events?